terragrunt/env/production/env_vars.hcl

```diff
@@ -1,9 +1,10 @@
 inputs = {
-  account_id        = "211125499457"
-  env               = "production"
-  product_name      = "cra_upd_dashboard"
-  cost_center_code  = "cra-upd-dashboard"
-  billing_code      = "CostCentre"
-  billing_tag_value = "CraUpdDashboard"
-  domain            = "cra-arc.cdssandbox.xyz"
+  account_id          = ""
+  env                 = "production"
+  product_name        = "cra_upd_dashboard"
+  product_name_dashed = "cra-upd-dashboard"
```

Comment:
Do you need to specify _dashed variable? Can you just call the product_name `cra-upd-dashboard` as this would eliminate the need for an extra variable.

Reply:
I wasn't sure if I should change it from what you had originally set it as, but yes I'll make this change.